app/assets/stylesheets/helper/_bootstrap_overrides.scss

```diff
@@ -11,4 +11,5 @@ $mark-colour: #faf785;
 mark,
 .mark {
```

Comment:
We probably don't, but thought it would be better if we override bootstrap properly in case someone adds a `.mark` and it's a different colour.